src/options.js

```diff
@@ -2,6 +2,5 @@ var options = {
   debug: false,
   updateRate: 15, // How often network components call `sync`
   compressSyncPackets: false, // compress network component sync packet json
-  useLerp: true, // when networked entities are created the aframe-lerp-component is attached to the root
```

Comment:
Getting rid of this flag is probably reasonable but there obviously needs to be a way to control this at the schema level. We should think about how that configuration will be extended later to support other interpolation models for other components.